src/components/TodoComponents/Todo.js

```diff
@@ -0,0 +1,31 @@
+
+import React from 'react';
+import './Todo.css'
+// import { list } from 'postcss';
+
+function Todo (props) {
+    // const{ toggleCompleted, lists } = props;
+
+    
+
+    
+    return (
+
+        <p 
+        
+        className={ 'thing.completed' ? 'completed' : null }
+        style={{ textDecorationLine: props.lists.completed ? 'line-through' : '' }}
+        onClick={ e => props.toggleCompleted(props.lists.id)}
+    
+        >
+        
+        {props.lists.task}
+         
+
+        </p>
+
+        
+    )
+}
+
+export default Todo;
```

Comment:
Looks good, but we should try to lessen the blank lines if we can... try using the Prettier extension on VSCode to automatically adjust your code, it will truncate the extra lines that happen while coding.